services/common/boltzmann/request-handler.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const { parse } = require('url');
```

Comment:
`url.parse` is deprecated, should this use `new URL` instead?

Reply:
Ah yeah. `new URL` has the (uncomfortable in this case) prereq that you have to give it two args to prevent it from throwing if the first arg is an "incomplete" url – we _probably_ should move over to it, but as a follow up PR.